src/Schema/Converter/SchemaToPHP/ArrayItemRenderer.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Schema\Converter\SchemaToPHP;
+
+final class ArrayItemRenderer
```

Comment:
I'd rename "renderer" to "exporter" similar to `var_export()`. Same for other related classes.

Reply:
Alternative is "writer".